src/Stratis.SmartContracts.Executor.Reflection/Loader/ContractByteCode.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Stratis.SmartContracts.Executor.Reflection.Loader
+{
+    /// <summary>
+    /// Represents a contract's bytecode.
+    /// </summary>
+    public class ContractByteCode
```

Comment:
Wouldn't block, but seems silly to me.

Interested in discussion: @fassadlr @majikandy